sys/Makefile

```diff
@@ -113,6 +113,12 @@ endif
 ifneq (,$(filter sixlowpan,$(USEMODULE)))
   DIRS += net/network_layer/sixlowpan
 endif
+ifneq (,$(filter gcoap,$(USEMODULE)))
+  DIRS += net/application_layer/gcoap
+endif
+ifneq (,$(filter gcoap_forward_proxy,$(USEMODULE)))
+  DIRS += net/application_layer/gcoap/forward_proxy
+endif
```

Comment:
I know, your PR is much older, but would it be too much of a bother to use the `SUBMODULE := 1` mechanism for the new submodule? This is the way I've introduced `gcoap_dns` in #16705.